mesop/web/src/component_renderer/component_renderer.ts

```diff
@@ -39,7 +38,7 @@ const WEB_COMPONENT_PREFIX = '<web>';
   templateUrl: 'component_renderer.ng.html',
   styleUrl: 'component_renderer.css',
   standalone: true,
-  imports: [CommonModule, ComponentLoader],
+  imports: [CommonModule],
```

Comment:
When was Component Loader used before? Thinking about it now, I don't actually remember looking at the file before.

Reply:
It hasn't been used in a very long time. At one point, I was deferring loading the components (e.g. lazy load only the components that you need) so I wrote ComponentLoader, but I couldn't actually get this working inside google3 because it compiles down into a dynamic import (e.g. `import(...)`) which has special handling in google3